bucket/dotnet-core-desktop-runtime.json

```diff
@@ -0,0 +1,22 @@
+{
+    "homepage": "https://dotnet.microsoft.com/download/dotnet-core",
+    "description": "Microsoft .NET Core Desktop Runtime",
+    "version": "3.1.5",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.microsoft.com/en-us/legal/intellectualproperty/copyright/default.aspx"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://download.visualstudio.microsoft.com/download/pr/86835fe4-93b5-4f4e-a7ad-c0b0532e407b/f4f2b1239f1203a05b9952028d54fc13/windowsdesktop-runtime-3.1.5-win-x64.exe",
+            "hash": "sha512:5df17bd9fed94727ec5b151e1684bf9cdc6bfd3075f615ab546759ffca0679d23a35fcf7a8961ac014dd5a4ff0d22ef5f7434a072e23122d5c0415fcd4198831",
+            "post_install": "Invoke-ExternalCommand -FilePath \"$dir\\windowsdesktop-runtime-3.1.5-win-x64.exe\" -ArgumentList \"/install /quiet /norestart\" -RunAs | Out-Null"
+        },
+        "32bit": {
+            "url": "https://download.visualstudio.microsoft.com/download/pr/df7b90d9-b93e-4974-85ef-c1de418bc186/e380e58bbd8505ebaee6c3abb23baade/windowsdesktop-runtime-3.1.5-win-x86.exe",
+            "hash": "sha512:b5e7e49d5b6f4f83a665d1f4c7d83700540f9c888c3c36ccd44f5395138dc1e258a4fcb260845d8ac7e3e5a1513be1666521edeaf6dfd67669a04a1290cda7e1",
+            "post_install": "Invoke-ExternalCommand -FilePath \"$dir\\windowsdesktop-runtime-3.1.5-win-x86.exe\" -ArgumentList \"/install /quiet /norestart\" -RunAs | Out-Null"
+        }
+    },
+    "notes": "You can now remove this installer with 'scoop uninstall dotnet-core-desktop-runtime'"

```

Comment:
```suggestion
    },
    "pre_install": "if (!(is_admin)) { error 'Admin privileges are required.'; break }",
    "installer": {
        "script": "Invoke-ExternalCommand \"$dir\\$fname\" -ArgumentList '/install', '/quiet', '/norestart' -RunAs | Out-Null"
    }
````

Reply:
Usually speaking a UAC window will pop up and ask for admin rights anyway. I assume if pre_install condition is added, and user didn't execute the install command with sudo or from escalated shell, they will need to do "scoop uninstall" then "scoop install"?

Reply:
...

https://scoop.sh/

> Say goodbye to permission pop-ups
> Scoop installs programs to your home directory by default. So you don’t need admin permissions to install programs, and you won’t see UAC popups every time you need to add or remove a program.